crowbar_framework/db/migrate/20170105095251_add_last_seen_to_node.rb

```diff
@@ -0,0 +1,6 @@
+class AddLastSeenToNode < ActiveRecord::Migration
+  def change
+    add_column :nodes, :last_seen, :datetime
```

Comment:
I would call it `seen_at` which is similar to the rails defaults like created_at or updated_at.